dbschema/product.esdl

```diff
@@ -0,0 +1,108 @@
+module default {
+  abstract type Product extending Engagement::Child {
+    overloaded engagement: LanguageEngagement;
+
+    scriptureReferences: Scripture::Collection;
+
+    multi mediums: Product::Medium;
+    multi purposes: Product::Purpose;
+    multi steps: Product::Step;
+    methodology: Product::Methodology;
+
+    describeCompletion: str;
+    placeholderDescription: str;
+    pnpIndex: int16;
+    progressTarget: int16;
+    progressStepMeasurement: Product::ProgressMeasurement;
+
+    #  createdAt: cal::local_date;  #TODO - is this desired here?  Already exists in Mixin::Timestamped?
+
+    #TODO - category := ??? - add this computed field here after migration
+  }
+}
+
+module Product {
+  type DirectScriptureProduct extending default::Product {
```

Comment:
I'd keep the products in the default module. Follows suit of the concretes of projects & engagements & producibles.